pkg/downloader/manager.go

```diff
@@ -85,6 +85,10 @@ type Manager struct {
 // If SkipUpdate is set, this will not update the repository.
 func (m *Manager) Build() error {
 	c, err := m.loadChartDir()
+	if c.Metadata.Dependencies == nil && m.Debug {
+		fmt.Print("No dependencies found to build")
+		return nil
```

Comment:
Should the early `return` here be done? It isn't clear to me the logging means code should also early return.

Reply:
My assumption was that if there are no dependencies to build we can simply return. Is there something we do even if there are no dependencies? I didn't see a chart.lock file being created.